### ISSUE TITLE
SELECT * FROM information_schema.innodb_trx fails with [1146] [42S02] Table 'mysql.innodb_trx' doesn't exist

### ISSUE THREAD

**Issue by mp911de** [mirromutth/r2dbc-mysql#45]
Issuing statements to tables of a different database/schema are failing that the table cannot be found in the current database.
Running the same SQL over JDBC works.

MySQL Server version: 5.7.22

```sql
SELECT * FROM information_schema.innodb_trx
```

Error:
```
io.r2dbc.spi.R2dbcBadGrammarException: [1146] [42S02] Table 'mysql.innodb_trx' doesn't exist)
```

Reproducer:

```java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;

import io.github.mirromutth.r2dbc.mysql.MySqlConnectionConfiguration;
import io.github.mirromutth.r2dbc.mysql.MySqlConnectionFactory;
import reactor.test.StepVerifier;

public class Reproducer {

	public static void main(String[] args) throws Exception{

		Connection connection = DriverManager
				.getConnection("jdbc:mysql://localhost/mysql", "root", "my-secret-pw");

		ResultSet resultSet = connection.createStatement()
				.executeQuery("SELECT * FROM information_schema.innodb_trx");
		resultSet.next();
		resultSet.close();

		MySqlConnectionConfiguration configuration = MySqlConnectionConfiguration.builder()
				.host("localhost").username("root")
				.password("my-secret-pw").database("mysql").build();

		MySqlConnectionFactory connectionFactory = MySqlConnectionFactory.from(configuration);

		connectionFactory.create().flatMapMany(it -> it
				.createStatement("SELECT * FROM information_schema.innodb_trx").execute())
				.flatMap(it -> it.map((row, rowMetadata) -> row.get(0)))
				.as(StepVerifier::create)
				.verifyComplete();
	}
}
```

**Comment by mirromutth** [comment]
The reason is caused by capabilities, again.

All unknown/unused feature flags have been removed.

**Comment by mirromutth** [comment]
It solved by PR #48 , target version is 0.3.0.M3.
